tests/lib/snaps/test-snapd-cifs-mount/snapcraft.yaml

```diff
@@ -0,0 +1,15 @@
+name: test-snapd-cifs-mount
+version: 1.0
+summary: Basic cifs-mount snap
+description: A basic snap which allows mounting and unmounting CIFS filesystems
+
+apps:
+  sh:
+    command: sh
+    plugs: [cifs-mount]
```

Comment:
Probably needs mount-observe, per comments.

Reply:
@niemeyer , thanks for reviewing this. I tried by adding the mount-observe plug but still having the same denial. testlog: https://paste.ubuntu.com/p/X56wt4CHrw/